src/controllers/destination.ts

```diff
@@ -127,11 +127,11 @@ export default class DestinationController {
       ctx.body = { output: [errResp] };
     }
     ControllerUtility.postProcess(ctx);
-    stats.histogram('dest_transform_output_events', ctx.body?.output?.length, {
-      destination,
-      version: 'v0',
-      ...metaTags,
-    });
+    // stats.histogram('dest_transform_output_events', ctx.body?.output?.length, {
+    //   destination,
+    //   version: 'v0',
+    //   ...metaTags,
+    // });
```

Comment:
Let's remove commented code, please.

Reply:
kept original code as @utsabc will raise pr soon to fix it

Reply:
That'll be available in the version history.